docs/configuration.md

```diff
@@ -746,6 +746,18 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.files.useFetchCache</code></td>
+  <td>true</td>
+  <td>
+    If file fetching should use local caching. The improves performance when running multiple
```

Comment:
I'd reword this slightly in order to match the style of similar boolean options.  How about this:

> If set to true (default), file fetching will use a local cache that is shared by executors that belong to the same application, which can improve task launching performance when running many executors on the same host.  If set to false, these caching optimizations will be disabled and all executors will fetch their own copies of files.  This optimization may be disabled in order to use NFS filesystems as Spark local directories (see <a href="https://issues.apache.org/jira/browse/SPARK-6313">SPARK-6313</a> for more details).